tools/scripts/validate_alerts.py

```diff
@@ -0,0 +1,39 @@
+import argparse
```

Comment:
A really nice BE task would be to bring lintrunner to `test-infra` I think :)

Reply:
agreed, after alerting / the caching stuff. I'll look into it if I have time this half.